ui-frontend/packages/catalog-ui-search/src/main/webapp/react-component/metacard-quality/container.tsx

```diff
@@ -46,13 +46,14 @@ class MetacardQuality extends React.Component<Props, State> {
   componentDidMount() {
     setTimeout(() => {
       const metacardId = this.model.get('metacard').get('id')
+      const storeId = this.model.get('metacard').get('properties').get('source-id')
 
       const attributeValidationRes = fetch(
-        `./internal/metacard/${metacardId}/attribute/validation`
+        `./internal/metacard/${metacardId}/${storeId}/attribute/validation`
```

Comment:
❓ Anywhere that's hitting this endpoint now will cause an error if the item is not in the local catalog. Should those other places be updated?

Reply:
Yea, I'm also a proponent of passing the source around with everything even if it is local. The assumption that it's local tends to burn us.